src/precice/impl/SolverInterfaceImpl.hpp

```diff
@@ -362,6 +365,29 @@ class SolverInterfaceImpl {
   /// Computes and maps all read data mapped to mesh with given ID.
   void mapReadDataTo(int toMeshID);
 
+  /**
+   * @brief Write vectorial data to the interface mesh
+   *
+   * The exact mapping and communication must be specified in XYZ.
+   *
+   * @param[in] fromDataID ID of the data to be written, e.g. 1 = forces
+   * @param[in] dataPosition Position (coordinate, e.g.) of data to be written
+   * @param[in] dataValue Value of the data to be written
+   */
+  void writeVectorData(
+      int           fromDataID,
+      int           valueIndex,
+      const double *value);
+
+  /**
+   * @copydoc precice::SolverInterface::writeVectorGradientData()
+   */
```

Comment:
```suggestion
  /// @copydoc precice::SolverInterface::writeVectorGradientData
```
You can remove the brackets and use a single-line comment. This is shorter and simpler with the same effect. See https://github.com/precice/precice/pull/1249. (You only have to provide the arguments of the function, if it is not unique to avoid ambiguities)

You can check the resulting doxygen by running `make doxygen` and then opening `precice/docs/source-code-documentation/index.html`